dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst/AppDelegate.cs

```diff
@@ -16,7 +16,7 @@ public override bool FinishedLaunching (UIApplication application, NSDictionary
 
 			// create a UIViewController with a single UILabel
 			var vc = new UIViewController ();
-			vc.View.AddSubview (new UILabel (Window.Frame) {
+			vc.View!.AddSubview (new UILabel (Window!.Frame) {
```

Comment:
uiVieController.View can be null? Weird

Reply:
Yep:

https://github.com/xamarin/xamarin-macios/blob/ef0a179610bdfe5e14bb7f18fdb4ed575746488c/src/uikit.cs#L14634-L14636